### PR TITLE
arp-74439 add ARP unified sign in onboarding

### PR DESCRIPTION
**Pull request by gabezurita**
@department-of-veterans-affairs/benefits-accredited-rep-facing, here's the [unified sign-in onboarding doc](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/74439) we were tasked with filling out. I'll add PR comments to guide feedback!

See [this Slack thread for context](https://dsva.slack.com/archives/CSFV4QTKN/p1706117746651949).

Once I get your feedback, I'll send it to the [#vsp-identity](https://dsva.slack.com/archives/CSFV4QTKN) team.

### CODE REVIEW

**Review by gabezurita — COMMENTED** [review]
@department-of-veterans-affairs/benefits-accredited-rep-facing, see below comments on where this onboarding doc needs feedback!

File: products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. **What is the name of the application?**
+
+   > Accredited Representative Portal (ARP)
+
+2. **What URL endpoints should redirect to your service?**
+
+   > Localhost: localhost:3000/representative/
+   > Dev: dev.va.gov/representative/
+   > Staging: staging.va.gov/representative/
+   > Production: www.va.gov/representative/
```
Comment (nihil2501) [inline products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:14]: we're still at `/representatives` (notice the s at the end)

File: products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Onboarding
+
+## 🏄 New application using the Unified Sign in Page questionnaire
+
+1. **What is the name of the application?**
+
+   > Accredited Representative Portal (ARP)
+
+2. **What URL endpoints should redirect to your service?**
```
Comment (nihil2501) [inline products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:9]: I actually don't understand this question.

File: products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   > PKCE
+
+5. **Which service providers are allowed to Sign up?**
+
+   > Login.gov
+
+6. **Which service providers can be used to Sign in?**
+
+   > Login.gov
+
```
Comment (nihil2501) [inline products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:33]: also ID.me

File: products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+7. **Does your service have user type requirements?**
```
Comment (nihil2501) [inline products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:34]: We'll need to figure out how our app presents authn and authz concepts relative to these things like identity verification and LOA.

**Review by nihil2501 — COMMENTED** [review]
@gabezurita Given that we will be pairing with identity on implementing our configuration changes, we may not be adding much value by landing answers to this preliminary questionnaire in our product documentation.

In general, we could consider closing this PR and proceeding to schedule our pairing sessions with identity as soon as possible.

File: products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md (hunk 1 above)
Comment (nihil2501) [reply products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:14]: Yes I think so.

Comment (nihil2501) [reply products/accredited-representative-facing/engineering/unified-sign-in-onboarding.md:14]: Another option is `accredited-representative-portal` for example.

**Pull request merged**